docker/prover/Dockerfile

```diff
@@ -1,11 +1,9 @@
-# For using private GitHub dependencies, CI downdloads all crates and bellman-cuda dependency outside of the contatiner
-# Not expected to work locally
-
-# syntax=docker/dockerfile:experimental
 FROM nvidia/cuda:11.8.0-devel-ubuntu22.04 as builder
 
 ARG DEBIAN_FRONTEND=noninteractive
 
+ARG ERA_BELLMAN_CUDA_RELEASE=latest
```

Comment:
ERA_BELLMAN_CUDA_RELEASE has only pre-releases, to support latest we should change release types there.